tests/queries/0_stateless/03168_query_log_privileges_not_empty.sh

```diff
@@ -1,32 +1,28 @@
 #!/usr/bin/env bash
-# Tags: no-parallel
 
 CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CUR_DIR"/../shell_config.sh
 
-user_name="u_03168_query_log"
-table_name="default.d_03168_query_log"
+user_name="u_03168_query_log_${CLICKHOUSE_DATABASE}"
```

Comment:
not sure I understand why tbh (unless flakyness check runs in parallel as well)

Reply:
Flaky check does run tests in parallel